lib/task-data/schemas/rackhd-task-schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "rackhd/schemas/v1/task-schema",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "RackHD Tasks Schema Extention",
+    "description": "The extension for RackHD tasks schema, all RackHD tasks schema should adhere to this schema.",
+    "allOf": [
+        {
+            "$ref": "http://json-schema.org/draft-04/schema"
+        }
+    ],
+    "definitions": {
+        "describeJob": {
+            "type": "string",
+            "description": "This property is required to point to a job's di injectable name."
```

Comment:
I think either is OK.
I see redfish schema has similar word, you can see the `readonly` description which is copied from redfish schema.